src/renderer/views/Watch/Watch.vue

```diff
@@ -122,7 +122,7 @@
       />
       <watch-video-chapters
         v-if="!hideChapters && !isLoading && videoChapters.length > 0"
-        :compact="backendPreference === 'invidious'"
+        :compact="!videoChapters[0].thumbnail"
```

Comment:
Will `!videoChapters[0].thumbnail` blow up if `videoChapters === []`?

Reply:
The v-if above will prevent that 🙂